lib/assets/components/explore_tiles.dart

```diff
@@ -0,0 +1,183 @@
+import 'package:app/assets/components/custom_network_image.dart';
+import 'package:app/models/Action.dart';
+import 'package:app/models/Campaign.dart';
+import 'package:app/models/Cause.dart';
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+
+class ExploreCampaignTile extends StatelessWidget {
+  final String headerImage;
+  final String title;
+  final ListCause cause;
+
+  ExploreCampaignTile(ListCampaign model, {Key? key})
+      : headerImage = model.headerImage,
+        title = model.title,
+        cause = model.cause,
```

Comment:
@JElgar I think you might've been looking at an older commit

Reply:
Oops!